src/components/Comments/WriteComment.js

```diff
@@ -90,7 +90,7 @@ const WriteComment = ({
           />
         </View>
       </View>
-      <View style={styles.container(inReplyCommentView)}>
+      <View style={styles.container}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a dynamic `paddingLeft` in `container` to a static value may affect the layout depending on the `inReplyCommentView` state. If this state is important for the layout, consider reverting this change.
```diff
-    paddingLeft: 20,
+    paddingLeft: inReplyCommentView ? 43 : 20,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->